.travis.yml

```diff
@@ -17,12 +17,9 @@ env:
     - JOB=test
 
 before_install:
-  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 0.28.1
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
 
 script:
-  # Babel doesn't build correctly with npm v2 (which comes bundled with Node v4)
-  # so we manually upgrade to a known working npm version.
-  - npm install --global npm@4
```

Comment:
I saw this so I thought we need to just use yarn for the installing for each package too